NEW_README.md

```diff
@@ -1,132 +1,308 @@
+
+In order to run the the software in this repo, you need the following
+requirements installed.
+
+* [Ruby](https://www.ruby-lang.org/en/)
+* [Bundler](https://bundler.io/)
+
+
+### Environment setup
 
 * Clone the gatling-puppet-load-test repo locally: https://github.com/puppetlabs/gatling-puppet-load-test
 * cd into the gatling-puppet-load-test root directory
-* For apples to apples runs, you can use the checked-in hosts files: [pe-perf-test.cfg](config/beaker_hosts/pe-perf-test.cfg) or [foss-perf-test.cfg](config/beaker_hosts/foss-perf-test.cfg). These are the defaults for the performance rake task based on the specified BEAKER_INSTALL_TYPE (pe or foss).
-* In order for us to take advantage of the new AWS account with access to internal network resources, you need to use [ABS](https://github.com/puppetlabs/always-be-scheduling). The 'performance' task will automatically use ABS to provision 2 AWS instances and then execute tests against those instances.
-* ABS requires a token when making requests. See the [Token operations](https://github.com/puppetlabs/always-be-scheduling#token-operations) section of the ABS README file for instructions to generate a token.
-Once generated, either set the ABS_TOKEN environment variable with your token or add it to the .fog file in your home directory using the abs_token parameter. For example:
+
+Several infrastructure variants are currently supported by the rake tasks in
+this repo.  The specific environment setup steps for each environment are
+outlined below.
+
+#### Environment variables
+
+The following environment variables are largly common to the rake tasks shown
+below.  They are ase follows.
+
+`BEAKER_INSTALL_TYPE`
+_(git, foss, pe)_ : Determines the underlying path structure of the
+puppet install.
+
+`BEAKER_OPTIONS_FILE`
+: Path to file containing additionally defined beaker options.
+
+`BEAKER_PE_VER`
+: The desired PE version to be installed. Must be available within
+the `BEAKER_PE_DIR`.
+
+`BEAKER_PE_DIR`
+: Path or URL where PE builds are stored for the `BEAKER_PE_VER`.
+
+
+It can be helpful to use an env file to manage these environment variables. The file
+[config/env/env_setup_2019.0.1](config/env/env_setup_2019.0.1) is provided as an example.
+
+Apply this configuration with the following command:
+```
+source config/env/env_setup_2019.0.1
+```
+
+For more information, see the beaker documentation on
+[beaker environment variables](https://github.com/puppetlabs/beaker/blob/master/docs/concepts/argument_processing_and_precedence.md#environment-variables)
+
+
+#### For AWS Execution
+
+[Create an AWS access key pair for your AWS account.](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/)
+Ensure that the key id and secret key are present in an appropriate section of
+your `$HOME/.fog` file.  Here is an example: _FIXME: Reference fog website._
+```
+:default:
+  :aws_access_key_id: <access key here>
+  :aws_secret_access_key: <secret key here>
+```
```

Comment:
We should add a bit to remind people like ourselves :)  that instances terminate in 24 hours... this can be controlled with ABS_AWS_REAP_TIME which defaults to 86400  which is 24 hours in seconds...  But it has to be set when the nodes are provisioned.

Reply:
I have added a note to this effect in the ABS section.